lgi/lgi.h

```diff
@@ -29,6 +29,29 @@
 #endif
 #endif
 
+/* Lua 5.3 integers. */
+#if LUA_VERSION_NUM >= 503
+// For Lua 5.3
+typedef LUA_UNSIGNED lgi_Unsigned;
```

Comment:
Why `LUA_UNSIGNED`? It does not appear in the API documentation of Lua 5.3. Only `lua_Unsigned` does. Also, the Lua 5.1 and Lua 5.2 cases use `lua_Unsigned`.

Reply:
I saw this in `luaconf.h`, so came with this. Ok, going to fix.